app/services/swiss_implementation.rb

```diff
@@ -64,11 +65,16 @@ def delta(a, b)
     end
 
     def targets(player)
-      players - [player] - excluded_opponents(player)
+      out = players - [player]
+
+      excluded = excluded_opponents(player)
+
+      out.reject { |p| excluded.include?(p == SwissImplementation::Bye ? p : p.id) }
```

Comment:
Is there a reason for this change?

Reply:
this was an artifact of changing the type for players.